modules/optimization/doc/content/source/optimizationreporters/ParameterMeshOptimization.md

```diff
@@ -36,6 +50,18 @@ The mesh is also used in the [ParameterMeshFunction.md] objects in the forward a
 
 The transfer of parameters is the same as in other inverse optimization problems, see [OptimizationReporter.md].
 
+## Example Input File Syntax For Restarting An Optimization Simulation
+
+A good guess for the initial parameter values will help with the convergence of the optimization problem.  In this example, initial conditions for the controllable parameters are given on an exodus mesh.  The parameter values on the exodus mesh are the optimized parameter values found in the previous example.  The parameter values from the above optimization problem are output by its forward problem onto the 10x10 mesh used by the FEM simulation.  The forward problem parameter field is then projected onto a coarser two-by-two parameter mesh as a first order Lagrange nodal AuxVariable called `restart_source` using [SolutionUserObject.md] and [SolutionAux.md] in the following input file
+
+!listing mesh_source/parameter_mesh_restart.i
+
+After the parameter mesh is initialized with parameter values, it can be used as an initial guess in the optimization problem.  The AuxVariable `restart_source` is then read in by `OptimizationReporter` using `initial_condition_mesh_variable` as the initial guess for the parameter field:  
+
+!listing mesh_source/main_linearRestart.i block=OptimizationReporter
+
+This optimization simulation is restarted with the optimized parameter values from the first example and convergence occurs in a single step.  The `parameterMeshOptimization` reporter also defined an `exodus_timesteps_for_parameter_mesh_variable` to identify which step to read from the exodus file.  Constant lower and upper bounds were also given by `constant_group_lower_bounds` and `constant_group_upper_bounds` but do not have an effect on the optimization algorithm because it was started from the exact solution for the parameter field.  To use bounds, the `tao_solver` in the `Executioner` block was changed to `taoblmvm` (bounded lmvm).  For difficult optimization problems, it is better to start with a good guess for the parameters than to tighten up their bounds.  Multiresolution optimization strategies [!citet](eslaminia2022full) find better initial guesses for the parameters by starting with an easier to solve optimization problem containing a few controllable parameters and then reusing those optimized parameters as an initial guess for a more difficult optimization problem with more parameters.  
```

Comment:
```suggestion
This optimization simulation is restarted with the optimized parameter values from the first example and convergence occurs in a single step.  The `ParameterMeshOptimization` reporter also defined an `exodus_timesteps_for_parameter_mesh_variable` to identify which step to read from the exodus file.  Constant lower and upper bounds were also given by `constant_group_lower_bounds` and `constant_group_upper_bounds` but do not have an effect on the optimization algorithm because it was started from the exact solution for the parameter field.  To use bounds, the `tao_solver` in the `Executioner` block was changed to `taoblmvm` (bounded lmvm).  For difficult optimization problems, it is better to start with a good guess for the parameters than to tighten up their bounds.  Multi-resolution optimization strategies [!citet](eslaminia2022full) find better initial guesses for the parameters by starting with an easier to solve optimization problem containing a few controllable parameters and then reusing those optimized parameters as an initial guess for a more difficult optimization problem with more parameters.  
```

Reply:
fixed